src/main/java/mate/academy/controllers/CreateCarController.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.lib.Injector;
+import mate.academy.model.Car;
+import mate.academy.model.Manufacturer;
+import mate.academy.service.CarService;
+import mate.academy.service.ManufacturerService;
+
+public class CreateCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final ManufacturerService manufacturerService = (ManufacturerService) injector
+            .getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/cars/create.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String model = req.getParameter("model");
+        Long manufacturerId = Long.valueOf(req.getParameter("manufacturerId"));
```

Comment:
```suggestion
        Long manufacturerId = Long.valueOf(req.getParameter("manufacturer_id"));
```
let's use the same approach everywhere for request parameters